### DISCUSSION TITLE
IBAX points

### DISCUSSION THREAD

**Discussion by hes91** [IBAX-io/go-ibax#891] [Q&A]
### Describe the bug

Does not give 200 points for liking and tagging 3 friends on Twitter posts.

### Reproduction

![A4DC34C8-7A0E-452F-981A-DC2EA0AF0F25](https://user-images.githubusercontent.com/98282930/150967683-ea96543d-915a-49c3-a463-cfe1742df551.png)
![54C3E878-2C52-42F6-A1BC-089764075F55](https://user-images.githubusercontent.com/98282930/150967698-3f4e8fbb-de05-4a72-89d8-da352a40adec.png)
![C845DDB5-7322-450E-9F0B-88178F755FDE](https://user-images.githubusercontent.com/98282930/150967707-26f06334-b16f-46ea-ab61-0a587d017770.png)
It does not give 200 points as it appears here

### System Info

```shell
All browsers have the same problem. A general problem.
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by 2558Mete** [comment]
Goof

**Comment by 2558Mete** [comment]
Nice

**Comment by 2558Mete** [comment]
aynen

**Comment by HalloKyle** [comment]
It's the original design users get 200 IBAX points.  Now we give more than 200

**Comment by Memix172** [comment]
Try send email to ibax team
